### PR TITLE
Opensource section addition :

### PR DESCRIPTION
**Pull request by paarth17032000**
- Add Opensource section and customized according to laptop and mobile
- Add Docs to navbar 
- Add github to social icons

### CODE REVIEW

File: components/Opensource/opensource-section.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <div
+      className={styles.main_area}
+      data-aos="fade-up"
+      data-aos-once="true"
+      data-aos-delay="300"
+    >
+      <h5>Opensource</h5>
+      <h1>Buidlfy is Open Source</h1>
+      <p>
+        There are many opportunities for designers, software engineers and
```
Comment (aayushmahapatra) [inline components/Opensource/opensource-section.tsx:15]: Update this to - "There are lots of opportunities for designers, developers, and enthusiasts to contribute to the Buidlfy ecosystem."

File: components/Opensource/opensource-section.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        enthusiasts to contribute to the Buidlfy ecosystem
+      </p>
+      <button
+        className={styles.gradient_button}
+        onClick={() => window.open("https://app.buidlfy.com", "_blank")}
```
Comment (aayushmahapatra) [inline components/Opensource/opensource-section.tsx:20]: This should open Github not the waitlist.

**Review by aayushmahapatra — CHANGES_REQUESTED** [review]

File: components/Opensource/opensource-section.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+function OpensourceSection() {
+  return (
+    <div
+      className={styles.main_area}
+      data-aos="fade-up"
+      data-aos-once="true"
+      data-aos-delay="300"
+    >
+      <h5>Opensource</h5>
```
Comment (rekpero) [inline components/Opensource/opensource-section.tsx:12]:
```suggestion
      <h5>We love OSS</h5>
```

**Review by rekpero — CHANGES_REQUESTED** [review]
a small content change

**Review by aayushmahapatra — APPROVED** [review]
lgtm

**Review by aayushmahapatra — APPROVED** [review]

**Review by rekpero — APPROVED** [review]

**Pull request merged**